Source/Diagnostics/FlushFormats/FlushFormatAscent.cpp

```diff
@@ -16,16 +16,25 @@ FlushFormatAscent::WriteToFile (
     amrex::Vector<amrex::Geometry>& geom,
     const amrex::Vector<int> iteration, const double time,
     const amrex::Vector<ParticleDiag>& particle_diags, int nlev,
-    const std::string prefix, int /*file_min_digits*/, bool plot_raw_fields,
+    const std::string prefix, int file_min_digits, bool plot_raw_fields,
     bool plot_raw_fields_guards,
     const bool /*use_pinned_pc*/,
-    bool /*isBTD*/, int /*snapshotID*/, const amrex::Geometry& /*full_BTD_snapshot*/,
+    bool isBTD, int snapshotID, const amrex::Geometry& /*full_BTD_snapshot*/,
```

Comment:
Ascent and SENSEI do not really yet support BTD. We should add an assert here.

Talk to me about BTD in situ vis, would make a good R&D project.

Reply:
ok, assert is added.  BTD in situ vis is reserved for future discussion :)